src/NuGet.Core/NuGet.Build.Tasks.Pack/MSBuildLogger.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NuGet.Common;
+
+namespace NuGet.Build.Tasks.Pack
+{
+    /// <summary>
+    /// TaskLoggingHelper -> ILogger
+    /// </summary>
+    internal class MSBuildLogger : Common.ILogger
+    {
+        private readonly TaskLoggingHelper _taskLogging;
+
+        public MSBuildLogger(TaskLoggingHelper taskLogging)
+        {
+            _taskLogging = taskLogging;
+        }
+
+        public void LogDebug(string data)
+        {
+            _taskLogging.LogMessage(MessageImportance.Low, data);
+        }
+
+        public void LogError(string data)
+        {
+            // Log errors as warnings. Then log the error summary as actual errors.
+            LogWarning(data);
```

Comment:
Why not log errors as errors?  That's what a caller of `LogError(...)` would expect to happen.